packages/components/src/tools-panel/stories/typography-panel.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalFontAppearanceControl as FontAppearanceControl,
+	__experimentalFontFamilyControl as FontFamilyControl,
+	__experimentalLetterSpacingControl as LetterSpacingControl,
+	LineHeightControl,
+} from '@wordpress/block-editor';
+import {
+	FontSizePicker,
+	__experimentalRadio as Radio,
+	__experimentalRadioGroup as RadioGroup,
+} from '@wordpress/components';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { ToolsPanel, ToolsPanelItem } from '..';
+import Panel from '../../panel';
+import { ControlLabel } from '../../ui/control-label';
+import { FormGroup } from '../../ui/form-group';
```

Comment:
Since we're already inside the `@wordpress/components` package, we can import these components via relative path, e.g.

```suggestion
/**
 * WordPress dependencies
 */
import {
	__experimentalFontAppearanceControl as FontAppearanceControl,
	__experimentalFontFamilyControl as FontFamilyControl,
	__experimentalLetterSpacingControl as LetterSpacingControl,
	LineHeightControl,
} from '@wordpress/block-editor';
import { useState } from '@wordpress/element';

/**
 * Internal dependencies
 */
import FontSizePicker from '../../font-size-picker'
import Radio from '../../radio';
import RadioGroup from '../../radio-group';
import { ToolsPanel, ToolsPanelItem } from '..';
import Panel from '../../panel';
import { ControlLabel } from '../../ui/control-label';
import { FormGroup } from '../../ui/form-group';
```